CONTRIBUTING.md

```diff
@@ -21,3 +21,32 @@ Luckily, if you're not willing to get too involved with the technicalities of Gi
 The [Article Styling Criteria (ASC)](https://github.com/ppy/osu-wiki/tree/master/wiki/Article_styling_criteria) is a set of standards that must be adhered to for all new and edited articles going forward. Among these are rules regarding project structure, usage of Markdown, formatting, diction and grammar. Please refer to it if you have any specific questions or concerns.
 
 The ASC is fairly verbose and encompasses many minute details, that are not necessary to memorise in order to take the first step of contributing to the osu! wiki. Have a look at the [contribution guide](https://github.com/ppy/osu-wiki/tree/master/wiki/osu!_wiki/Contribution_guide), and other contributors will also be happy to answer your questions in the `#osu-wiki` channel of the [osu!dev Discord server](https://discord.gg/ppy).
+
+## Testing
+
+All pull requests are automatically run through a series of continuous integration checks (CI) in order to catch various common errors.
+
+For people familiar with the command line, these CI checks can also be run locally.
+
+### Requirements
+
+- Python 3.x
+- Node.js 16
```

Comment:
Taking it to the next level would be to just use docker, avoiding all of the venv and dependency nonsense. Can be done as a future effort if that's easier though.

Reply:
docker is overkill while separate environments work just fine (on the other hand we wouldn't probably even need that if all of our CI was in Node.js)

Reply:
It's not overkill if it makes running 10x easier on platforms like windows. But sure.

Reply:
this really depends on how much we want to support bare Windows instead of recommending people use subsystems like Cygwin or WSL (which is notoriously easy to set up). if we want to simplify things as much as possible, then Docker may be an eventual good solution

Reply:
I personally have an aversion to running things like node or python on my main development system due to security concerns, and will only run it within docker or a VM.